pymobiledevice3/restore/restore.py

```diff
@@ -144,11 +168,14 @@ def send_filesystem(self, message: Mapping):
             self.logger.info('sending filesystem now...')
             asr.send_payload(filesystem)
 
+        # asr.close()
+
     def get_build_identity_from_request(self, msg):
         return self.get_build_identity(msg['Arguments'].get('IsRecoveryOS', False))
 
-    def send_buildidentity(self, message: Mapping):
+    def send_buildidentity(self, message: Mapping) -> None:
         self.logger.info('About to send BuildIdentity Dict...')
+        service = self._get_service_for_data_request(message)
         req = {'BuildIdentityDict': dict(self.get_build_identity_from_request(message))}
         arguments = message['Arguments']
         variant = arguments.get('Variant')
```

Comment:
`req['Variant'] =  arguments.get('Variant','Erase')` instead of if else?

Reply:
done